openpype/modules/default_modules/sync_server/providers/sftp.py

```diff
@@ -323,7 +331,8 @@ def delete_file(self, path):
         if not self.file_path_exists(path):
             raise FileNotFoundError("File {} to be deleted doesn't exist."
                                     .format(path))
-        self.conn.remove(path)
+        conn = self._get_conn()
```

Comment:
Shouldn't this be just `self.conn.remove(path)` as `conn` is property that will do that?